specs/dotsigstore_bundle/verification_flows.md

```diff
@@ -0,0 +1,38 @@
+# Purpose
+
+Enumerate the cosign verification use-cases with various keyed/keyless
+modes and TUF configurations. Scneraios with different Rekor/Fulcio
+instances (e.g. local, staging, corporate etc) are not explicitly
+listed, as via the correct command line parameter, the client should
+be able to figure out which keys to load in the TUF repository.
+
+| Case | Keytype | Rekor entry | Root CA | Key status | Verification passes |
+|-------------|---------|-------------|---------|------------|--------------|
+| #1   | Keyless | Yes         | Fulcio  | Valid      | :white_check_mark:  |
+| #2   | User provided | No | N/A | N/A: Verifier provides a set of accepted keys | :white_check_mark: |
+| #2.1 | User provided | Yes | N/A | Not known by verifier | :no_entry_sign: |
```

Comment:
What's the point of this row? To illustrate that the bundle-provided key ID isn't sufficient for verification?

Reply:
Yes, to make sure a verification must not succeed if the public key is not known to the verifier. An example could be that the `hint` in the public key attribute is the PEM encoded public key, and so the verify must not use that to verify the signature.